src/ripple/rpc/handlers/AccountLines.cpp

```diff
@@ -93,9 +93,12 @@ Json::Value doAccountLines (RPC::Context& context)
     std::string strIdent (params[jss::account].asString ());
     AccountID accountID;
 
-    // Intentional assignment in if.  Extra parentheses silence warning.
-    if ((result = RPC::accountFromString (accountID, strIdent)))
+    if (auto jv = RPC::accountFromString (accountID, strIdent))
+    {
+        for (auto it = jv.begin (); it != jv.end (); ++it)
```

Comment:
The problem with range-for is that you don't get the iterator. I'm not sure if the body of that loop needs it though. I think this is solved in the Range TS?